tests/src/reloadFunctionalTest/kotlin/org/jetbrains/compose/reload/tests/ResourcesTests.kt

```diff
@@ -31,45 +32,61 @@ import kotlin.io.path.name
 @ExtendBuildGradleKts(ResourcesTests.Extension::class)
 class ResourcesTests {
```

Comment:
Thanks for adding those tests 😍
Do you think there will be issues when people switch from regular strings to plurals?

Reply:
Do you suggest to add a test? I don't expect problems here.